internal/testutil/servicetest/orchestratortest/orchestrator.go

```diff
@@ -11,8 +11,21 @@ import (
 	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
-// NewCertificate creates a mock certificate
-func NewCertificate() *orchestrator.Certificate {
+type NewCertificateOption func(*orchestrator.Certificate)
+
+func WithMockCertificateID(id string) NewCertificateOption {
+	return func(certificate *orchestrator.Certificate) {
+		certificate.Id = id
+	}
+}
+func WithMockServiceID(cloudServiceID string) NewCertificateOption {
+	return func(certificate *orchestrator.Certificate) {
+		certificate.CloudServiceId = cloudServiceID
+	}
+}
+
+// NewCertificate creates a mock certificate.
+func NewCertificate(opts ...NewCertificateOption) *orchestrator.Certificate {
```

Comment:
- [ ] I will probably replace the `NewCertificate` function with 2 mock certificate variables (directly)